multimedia/minisatip/Makefile

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (C) 2010-2012 OpenWrt.org
```

Comment:
You can't/shouldn't assign copyright to OpenWrt.org unless you're affiliated, either use your own or omit.

Reply:
@diizzyy If this is true, please update CONTRIBUTING.md so others don't do the same:

Makefile contents should contain:
An up-to-date copyright notice. **Use OpenWrt if no other present** or supply your own.